content/en/dashboards/reporting.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Reporting
+kind: documentation
+disable_toc: false
+is_public: true
+further_reading:
+  - link: "https://www.datadoghq.com/blog/dashboard-sharing/"
+    tag: "Blog"
+    text: "Share Datadog dashboards securely with anyone" 
+  - link: "https://www.datadoghq.com/blog/template-variable-associated-values/"
+    tag: "Blog"
+    text: "Use associated template variables to refine your dashboards"
+  - link: "https://learn.datadoghq.com/course/view.php?id=8"
+    tag: "Learning Center"
+    text: "Building Better Dashboards"
```

Comment:
This is a link to the old learning center and 404's

Reply:
Thanks for catching that, updated!